### PR TITLE
update StorytellerSDK to 9.1.2

### PR DESCRIPTION
**Pull request by bartoszstelmaszuk**

### CODE REVIEW

File: yarn.lock
Comment (stodirascu) [inline yarn.lock:1]: This shouldn't be present in git? (similar to a podfile.lock)

**Review by stodirascu — APPROVED** [review]

File: android/app/src/main/java/com/storytellerreactnative/MainActivity.java [hunk 1]
```diff
@@ -12,4 +15,22 @@ public class MainActivity extends ReactActivity {
...
+  /**
+   * Returns the instance of the {@link ReactActivityDelegate}. Here we use a util class {@link
+   * DefaultReactActivityDelegate} which allows you to easily enable Fabric and Concurrent React
+   * (aka React 18) with two boolean flags.
+   */
+  @Override
+  protected ReactActivityDelegate createReactActivityDelegate() {
+    return new DefaultReactActivityDelegate(
+        this,
+        getMainComponentName(),
```
Comment (fnabrdal) [inline android/app/src/main/java/com/storytellerreactnative/MainActivity.java:29]: Did you dev test Android?

File: yarn.lock
Comment (fnabrdal) [reply yarn.lock:1]: Is it similar to `package-lock.json` ? If so it should stay.

File: android/app/src/main/java/com/storytellerreactnative/MainActivity.java (hunk 1 above)
Comment (bartoszstelmaszuk) [reply android/app/src/main/java/com/storytellerreactnative/MainActivity.java:29]: Yes, I run it through terminal and android studio :+1:

**Pull request merged**